scripts/document.sh

```diff
@@ -53,7 +43,7 @@ find Sources/Mapbox{Core,}Navigation/ -name '*.swift' -exec \
 find Sources/Mapbox{Core,}Navigation/ -name '*.[hm]' -exec \
     perl -pi -e 's/([<"])MapboxCoreNavigation\b/$1MapboxNavigation/' {} \;
 
-jazzy \
+bundle exec jazzy \
```

Comment:
Nice, I guess we could even simplify this documentation (which is for developers who want to preview the generated documentation):

https://github.com/mapbox/mapbox-navigation-ios/blob/5382dc06ef592805dc17689ace4be423aa6d8ee9/docs/README.md#L3

Reply:
Yeah, now we need to require Bundler in this instruction.